src/blocks/subscribe/view.js

```diff
@@ -13,37 +15,86 @@ import './style.scss';
 			const messageContainer = container.querySelector(
 				'.newspack-newsletters-subscribe-response'
 			);
+			const getCaptchaToken = () => {
+				return new Promise( ( res, rej ) => {
+					const reCaptchaScript = document.getElementById( 'newspack-recaptcha-js' );
+					if ( ! reCaptchaScript ) {
+						return res( '' );
+					}
+
+					const { grecaptcha } = window;
+					if ( ! grecaptcha ) {
+						return res( '' );
+					}
+
+					const captchaSiteKey = reCaptchaScript.getAttribute( 'src' ).split( '?render=' ).pop();
+					if ( ! grecaptcha?.ready || ! captchaSiteKey ) {
+						rej( __( 'Error loading the reCaptcha library.', 'newspack-newsletters' ) );
+					}
+
+					grecaptcha.ready( async () => {
+						const token = await grecaptcha
```

Comment:
There's still `async/await` here

Reply:
Bah, sorry for the back and forth. b75f03a should fix it once and for all.